Formula/libprelude.rb

```diff
@@ -0,0 +1,52 @@
+class Libprelude < Formula
+  desc "Universal Security Information & Event Management (SIEM) system"
+  homepage "https://www.prelude-siem.org/"
+  url "https://www.prelude-siem.org/attachments/download/1395/libprelude-5.2.0.tar.gz"
+  sha256 "187e025a5d51219810123575b32aa0b40037709a073a775bc3e5a65aa6d6a66e"
+  license "GPL-2.0-or-later"
+
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "gnutls"
+  depends_on "libgpg-error"
+  depends_on "python@3.8"
```

Comment:
No python@3.9?

Reply:
```
checking for /usr/local/opt/python@3.9/bin/python3... no
checking for python3.8... no
checking for python3.7... no
checking for python3.6... no
checking for python3.5... no
checking for python3.4... no
checking for python3.3... no
checking for python3.2... no
checking for python3.1... no
checking for python3.0... no
configure: error: Could not find /usr/local/opt/python@3.9/bin/python3 binary
```

It's looking for python3, but not for the latest version. There is some activity upstream; so they might add it when they have time. Python 3.8 is good until 2024.